public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -39,6 +39,35 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 	</div>
 </div>
 
+<b>Connection Certificate Files</b>
+
+<div class="gf-form-group">
```

Comment:
Having discussed with my UI colleagues, we think these settings should be hidden if SSL mode is "disable".

Reply:
We also think that the forms contained in this form group, should be co-located with the "SSL Mode" form instead.